modules/core-testing/build.gradle.kts

```diff
@@ -1,15 +1,16 @@
 plugins {
     id("com.android.library")
-    id("org.jetbrains.kotlin.android")
+    kotlin("android")
 }
 
 @Suppress("UnstableApiUsage")
 android {
-    compileSdk = 32
+    namespace = "soy.gabimoreno.core.testing"
+    compileSdk = 33
 
     defaultConfig {
         minSdk = 23
-        targetSdk = 32
+        targetSdk = 33
```

Comment:
Do we need to update the `targetSdk` in this PR?

Reply:
It was upgraded and tested to remove this warning, but I am happy to return it to `32` if you prefer. 
```
Not targeting the latest versions of Android; compatibility modes apply. Consider testing and updating this version. Consult the android.os.Build.VERSION_CODES javadoc for details.
```

Reply:
Also in modules, it looks like targeting SDK is not required anymore, would you like to I delete them in this PR too?
![Screenshot 2023-06-19 at 21 39 22](https://github.com/soygabimoreno/Base/assets/43986561/f69664ed-2120-4102-a14e-7b40f9521f3c)

Reply:
I was suggesting 🤣.
As I need to make another commit to removing generated files for ios, is ok for you if I remove them?

Reply:
Sure!